app/_src/policies/meshservice.md

```diff
@@ -0,0 +1,90 @@
+---
+title: MeshService
+---
+
+{% warning %}
```

Comment:
I think we need a link to configuration how to enable this feature

Reply:
added